src/server/rpc/procedures/roboscape/roboscape.js

```diff
@@ -34,9 +46,9 @@
 const logger = require('../utils/logger')('roboscape');
 var dgram = require('dgram'),
     server = dgram.createSocket('udp4'),
-    NetworkTopology = require('../../../network-topology'),
-    FORGET_TIME = 120, // forgetting a robot in seconds
-    RESPONSE_TIMEOUT = 200, // waiting for response in milliseconds
+    SocketManager = require('../../../socket-manager'),
+    FORGET_TIME = 10, // forgetting a robot in seconds
+    RESPONSE_TIMEOUT = 700, // waiting for response in milliseconds
```

Comment:
do you need to change these two timeout values? it might have side effects

Reply:
there was a longer delay with the python robots and I had to up the timeout.